powersimdata/design/investment/investment_costs.py

```diff
@@ -45,8 +45,10 @@ def calculate_ac_inv_costs(scenario, sum_results=True, exclude_branches=None):
     NEEM regions are used to find regional multipliers.
 
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance.
-    :param bool sum_results: sum data frame for each branch type.
-    :return: (*dict*) -- cost of upgrading branches in $2010.
+    :param bool sum_results: whether to sum data frame for each branch type.
+    :return: (*dict*) -- keys are {'line_cost', 'transformer_cost'}, values are either
+        float if sum_results, or data frames indexed by branch ID, with a 'Cost' column
```

Comment:
Cost --> cost (from what I can see in the code)

Reply:
Also, you can \`\`sum_results\`\` here

Reply:
Good catch, I had forgotten to change this when I changed the function.

Reply:
Fixed.